01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,12 @@
+(1..50).each do |a|
+    if a%15==0
```

Comment:
```suggestion
  if a % 15 == 0
```

「インデントはスペース2文字」「演算子の前後にはスペースを入れる」というコーディングスタイルが一般的です。